test/uk/gov/hmrc/customs/emailfrontend/model/EmailDetailsSpec.scala

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.emailfrontend.model
+
+import org.scalatest.matchers.should.Matchers._
+import org.scalatestplus.play.PlaySpec
+import play.api.libs.json.Json
+
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class EmailDetailsSpec extends PlaySpec {
+
+  "serialize and deserialize to/from JSON properly for correct datetime value in EmailDetails" in new Setup {
```

Comment:
Its better to use the text, Reads and Writes instead of Serilaization and Deserialization. Same for other places